lib/server-impl.js

```diff
@@ -37,6 +38,7 @@ function createApp (options) {
 
 function start (opts) {
     const options = createOptions(opts);
+    process.env.TZ = options.timezone;
```

Comment:
Might be better to document that `TZ` will be read, and default to UTC? If the user specifies something else, that should be respected